### PR TITLE
Vpn guide

### PR DESCRIPTION
**Pull request by andrewawagner**
Adding Stratum VPN Appliance Guide to docs

### CODE REVIEW

File: guides/vpn-client-setup.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+**OSX 10.9+**
+  * Native VPN Client
+
+**Ubuntu/RedHat Linux**
+  * StrongSwan VPN Client 5.1+
+
+# VPN Security
+
+Communications between the your client computer and the Stratum environment over the VPN are encrypted and secure. However, your VPN credentials provide access into this protected environment and should be safeguarded very securely. Please make sure that you are following
```
Comment (josomo) [inline guides/vpn-client-setup.md]: following [...].

Comment (andrewawagner) [inline guides/vpn-client-setup.md]: Fixed this!

**Comment by jault3** [comment]
LGTM

**Comment by andrewawagner** [comment]
Added code example to service map and example. Credit to beach for photos in this article!

**Pull request merged**